### ISSUE TITLE
Windows: Runtime Not Found

### ISSUE THREAD

**Issue by JohnAlexCO** [servo/servo#28900]
Upon installation and trying to run from the installer, it just says `VCRUNTIME140_1.dll was not found`
![image](https://user-images.githubusercontent.com/81481181/175750126-5feb21e5-11a0-41eb-94e4-1ead4f470994.png)

**Comment by michaelgrigoryan25** [comment]
Could you please comment on whether or not you are still experiencing this issue, @JohnAlexCO?

**Comment by JohnAlexCO** [comment]
I wanted to test out if this issue still exists for you, so. Just downloaded the `servo-latest` binary installer to try and see if the issue still exists. Ran the installer, go to see if it works and... well, totally new issue: installer says its done, but the folder its supposed to have installed to is empty? Ran the uninstaller and tried again. Still empty. So, I'm not really sure if the issue still exists. The installer knows whether it's "installed" or not, so it clearly has side-effects of some kind even though it _appears_ to have done nothing at all.

Decide to try a default installation. Installer does not say where this program is now located, so I check program files. I guess it installs itself as "Servo Tech Demo"? It doesn't have this same DLL error anymore, but it doesn't run properly: it opens a webpage, tries to render it, and then **immediately** crash with no error message or log.

![image](https://user-images.githubusercontent.com/81481181/210410334-49b86f19-e076-4520-be98-d45dbdd1d3f9.png)
![image](https://user-images.githubusercontent.com/81481181/210410433-c20ace3f-bc8a-4b8d-98fe-d995e821ac19.png)
![image](https://user-images.githubusercontent.com/81481181/210411446-933e38a3-4315-4748-ac57-104245b40e3f.png)

**Comment by mrego** [comment]
This has happened to some people with the 0.0.1 release (CC @jschwe).

**Comment by jschwe** [comment]
> This has happened to some people with the 0.0.1 release (CC @jschwe).

This issue contains multiple error descriptions. Are you referring to the first one? Probably, it would make sense to test future releases in a VM with a clean windows install - I obviously ran `./mach bootstrap` before on my usual machine.

**Comment by mrego** [comment]
> > This has happened to some people with the 0.0.1 release (CC [@jschwe](https://github.com/jschwe)).
> 
> This issue contains multiple error descriptions. Are you referring to the first one? Probably, it would make sense to test future releases in a VM with a clean windows install - I obviously ran `./mach bootstrap` before on my usual machine.

Yes I was referring to the first error about `VCRUNTIME140_1.dll`, it has happened on a Windows 10 fresh installation at least.
